### DISCUSSION TITLE
About installing StreamLink on the Router

### DISCUSSION THREAD

**Discussion by zxctw** [streamlink/streamlink#4678] [General]
I want to try to install StreamLink on a Router running openWRT, Because I want to stream Twitch live through a player like VLC. I'm not sure if this is possible?

**Comment by bastimeyer** [comment]
If [`--player-external-http`](https://streamlink.github.io/cli.html#cmdoption-player-external-http) doesn't suit your needs, you'll probably have to write your own stream-serving logic in Python, or use the [`--stdout`](https://streamlink.github.io/cli.html#cmdoption-stdout) option and pipe it into another application like FFmpeg for example and make the stream available on your network device like that. Either way, what you're trying to achieve is a very niche thing and it's not the focus of Streamlink's CLI.

**Comment by mkbloke** [comment]
Maybe this will do what you want: https://github.com/back-to/liveproxy

**Reply by zxctw** [reply]
ok, I'll test it out

**Reply by zxctw** [reply]
@mkbloke  Hi, I'm wondering what should I type in VLC to stream?
